terraform/workspace_variables/review.backend.tfvars

```diff
@@ -0,0 +1,3 @@
+storage_account_name = "s165d01afqtstfstatedv"
+
```

Comment:
Can we add a comment like "The key is provided dynamically for each review app via the Makefile"